students/smikulec/module-2-html/exercises/03-recipe/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Apple pie recipe</title>
+    </head>
+    <body>
+        <h1>Apple Pie</h1>
+        <img src="apple-pie.jpg" width="450" height="350">
+        <h2>Total Time</h2>
+            <p>Prep: 20 min. Bake: 45 min.</p>
+        <h2>Makes</h2>
+            <p>8 servings</p>
+        <h2>Ingredients</h2>
+            <ul>
+                <li>1/2 cup sugar</li>
+                <li>1/2 cup packed brown sugar</li>
+                <li>3 tablespoons all-purpose flour</li>
+                <li>1 teaspoon ground cinnamon</li>
+                <li>1/4 teaspoon ground ginger</li>
+                <li>1/4 teaspoon ground nutmeg</li>
+                <li>6 to 7 cups thinly sliced peeled tart apples</li>
+                <li>1 tablespoon lemon juice</li>
+                <li>Pastry for double-crust pie (9 inches)</li>
+                <li>1 tablespoon butter</li>
+                <li>1 large egg white</li>
+                <li>Additional sugar</li>
+            </ul>
+        <h2>Directions</h2>
+            <ol>
+                <li>In a small bowl, combine the sugars, flour and spices; set aside. In a large bowl, toss apples with lemon juice. Add sugar mixture; toss to coat.</li>
+                <br>
+                <li>Line a 9-in. pie plate with bottom crust; trim even with edge. Fill with apple mixture; dot with butter.Roll remaining crust to fit top of pie; place over filling. Trim, seal and flute edges. Cut slits in crust.</li>
+                br>
```

Comment:
Samo `<li>` element može biti dijete `<ol>` i `<ul>` elemenata. Uz to, ovdje `<br>` nije ni ispravno otvoren.

Reply:
Ok, to sam isto kasnije vidjela da validator izbacuje kao grešku. :D